pb/config/config.proto

```diff
@@ -607,6 +604,14 @@ message DashboardTabAlertOptions {
 
   // Custom text to show for the debug link.
   string debug_message = 7;
+
+  // Wait time between emails. If unset or zero, an email will be sent only once
+  // it becomes a consistent failure, and not again until it succeeds.
+  // TestGrid does not pester about staleness
```

Comment:
nit: Maybe clarify "Regardless of how this is set, TestGrid does not pester about staleness" ?

Reply:
Also is this true?

Reply:
I think this is true in the current implementation. Is it supposed to be different?

Reply:
Nope, think that's fine, just wasn't sure. Carry on!